source/robot_model/include/robot_model/Model.hpp

```diff
@@ -1,18 +1,18 @@
 #pragma once
 
-#include <iostream>
-#include <string>
-#include <vector>
 #include <OsqpEigen/OsqpEigen.h>
-#include <pinocchio/parsers/urdf.hpp>
-#include <pinocchio/multibody/data.hpp>
+#include <iostream>
 #include <pinocchio/algorithm/crba.hpp>
 #include <pinocchio/algorithm/rnea.hpp>
+#include <pinocchio/multibody/data.hpp>
+#include <pinocchio/parsers/urdf.hpp>
 #include <state_representation/parameters/Parameter.hpp>
 #include <state_representation/parameters/ParameterInterface.hpp>
 #include <state_representation/robot/Jacobian.hpp>
 #include <state_representation/robot/JointState.hpp>
 #include <state_representation/space/cartesian/CartesianState.hpp>
+#include <string>
```

Comment:
standard / inbuilt libraries should be at the top of the list

for example, <string> ..., then <Eigen> / <pinocchio> ..., then <state_representation/xyz> ....

Same goes for the .cpp file